2017/09/28 21:25:56 [crit] 31119#31119: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET /icons/ubuntu-logo.png HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/icons/ubuntu-logo.png", host: "rajputdataserver.info", referrer: "http://rajputdataserver.info/"
2017/09/28 21:25:56 [error] 31119#31119: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET /icons/ubuntu-logo.png HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/icons/ubuntu-logo.png", host: "rajputdataserver.info", referrer: "http://rajputdataserver.info/"
2017/09/28 21:25:58 [crit] 31119#31119: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:25:58 [error] 31119#31119: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:26:00 [crit] 31119#31119: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:26:00 [error] 31119#31119: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:26:07 [crit] 31119#31119: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:26:07 [error] 31119#31119: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:29:08 [crit] 31216#31216: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:29:08 [error] 31216#31216: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:29:10 [crit] 31216#31216: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:29:10 [error] 31216#31216: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:29:12 [crit] 31216#31216: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:29:12 [error] 31216#31216: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:29:14 [crit] 31216#31216: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:29:14 [error] 31216#31216: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:40:20 [crit] 31485#31485: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:40:20 [error] 31485#31485: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:40:22 [crit] 31485#31485: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:40:22 [error] 31485#31485: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:40:24 [crit] 31485#31485: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:40:24 [error] 31485#31485: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:40:26 [crit] 31485#31485: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:40:26 [error] 31485#31485: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:40:28 [crit] 31485#31485: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:40:28 [error] 31485#31485: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:44:30 [crit] 31578#31578: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:44:30 [error] 31578#31578: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:44:31 [crit] 31578#31578: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:44:31 [error] 31578#31578: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:44:33 [crit] 31578#31578: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:44:33 [error] 31578#31578: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:44:36 [crit] 31578#31578: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:44:36 [error] 31578#31578: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
2017/09/28 21:44:45 [crit] 31578#31578: *1 connect() to unix:/root/py3djenv/run/gunicorn.sock failed (13: Permission denied) while connecting to upstream, client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock:/", host: "rajputdataserver.info"
2017/09/28 21:44:45 [error] 31578#31578: *1 open() "/root/justdial_data_extractor/statics/500.html" failed (13: Permission denied), client: 27.131.12.6, server: rajputdataserver.info, request: "GET / HTTP/1.1", upstream: "http://unix:/root/py3djenv/run/gunicorn.sock/", host: "rajputdataserver.info"
